index.js

```diff
@@ -310,7 +310,7 @@ module.exports = function (opts) {
   opts = opts || {}
 
   return new UI({
-    width: (opts || {}).width || 80,
+    width: (opts || {}).width || process.stdout.columns || 80,
```

Comment:
I think some folks run yargs in the browser, so `process`/`process.stdout` might not exist. Let's pull this into a helper method that looks something like this:

```js
function defaultWidth () {
  width = null
  if (process && process.stdout && process.stdout.columns) width = process.stdout.columns
  return width
}
```